benchmarking/repeat_detection_evaluator.py

```diff
@@ -0,0 +1,215 @@
+"""
+This file takes a path of an unprocessed map, maps out the repeated detections of a tag,
```

Comment:
It seems like there are some scripts that do almost the same thing. Is there a way to combine these in any way?